Rhinoceros_Engine/Convert/ToRhino.cs

```diff
@@ -146,7 +147,15 @@ public static RHG.Transform ToRhino(this BHG.TransformMatrix bhTrans)
 
             return rhTrans;
         }
+        
+        /***************************************************/
+
+        public static RHG.Point3d ToRhino(this BH.oM.Graphics.Vertex point)
```

Comment:
I think this conversion does not take all the information about the `Vertex` object to rhino. 
Converting back to BHoM would mean losing information.

I'd suggest to either:
A. Find another `Rhino.Geometry` object that can hold the `Color` property
B. Find a way to store the `Color` property
C. Make the method `private`

Reply:
This conversion does not store the colour information but the `RenderMesh` ones does, will a `Vertex` in Rhino exist outside of the context of a mesh?  I guess so since the lack of this function was the initial error message, but the information could be retrieved trough the rhino coloured mesh if that is the only way we're going to use this. 

Does that count as B? Not really fool proof if someone makes a vertex outside a mesh but not sure if that is/should be possible

Reply:
We can say that the conversion never happens outside the `ToBHoM(RenderMesh)` because we haven't found a use case yet, but it may exist.

The `Convert` should be consistent with itself, regardless of the `ToBHoM(RenderMesh)` so I think C. might be the best option at this point.

Reply:
Ok, will do